### PR TITLE
Add patcher utility

### PR DESCRIPTION
**Pull request by JanTrueno**
Add patcher utility.

### CODE REVIEW

File: PortMaster/utils/patcher.txt
Comment (kloptops) [inline PortMaster/utils/patcher.txt:1]: This file shouldn’t have a port header, like the love runtime you can assume it has `$controlfolder` and all the associated variables that comes with sourcing control.txt and device_info.txt. :)

**Review by kloptops — CHANGES_REQUESTED** [review]

File: PortMaster/utils/patcher.txt
Comment (JanTrueno) [reply PortMaster/utils/patcher.txt:1]: Thanks, I knew something fixed my issue. Should have checked what I could remove ^^

File: PortMaster/utils/patcher.txt [hunk 1]
```diff
@@ -1,17 +1,5 @@
...
-  controlfolder="/opt/tools/PortMaster"
-elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
-  controlfolder="$XDG_DATA_HOME/PortMaster"
-else
-  controlfolder="/roms/ports/PortMaster"
-fi
-
 # Source PM info
 source $controlfolder/control.txt
 source $controlfolder/device_info.txt
```
Comment (kloptops) [inline PortMaster/utils/patcher.txt:5]: these too 😁

**Review by kloptops — CHANGES_REQUESTED** [review]

File: PortMaster/utils/patcher.txt (hunk 1 above)
Comment (JanTrueno) [reply PortMaster/utils/patcher.txt:5]: I need those to get the $controlfolder, unless im doing something wrong somewhere else.

Comment (kloptops) [reply PortMaster/utils/patcher.txt:5]: This is run by the port script, no?

```sh
PATCHER_FILE="$GAMEDIR/patcher.txt"
PATCHER_GAME="Blag Machine"
PATCHER_TIME="about 7 years"

source "$controlfolder/utils/patcher.txt"
```

**Review by kloptops — APPROVED** [review]

**Pull request merged**